WireAnalytics/Sources/WireAnalytics/AnalyticsManager.swift

```diff
@@ -53,15 +57,42 @@ public struct AnalyticsManager: AnalyticsManagerProtocol {
     /// - Returns: An object conforming to AnalyticsSessionProtocol for the new session.
     public func switchUser(_ userProfile: AnalyticsUserProfile) -> any AnalyticsSessionProtocol {
         analyticsService.endSession()
-        analyticsService.changeDeviceID(userProfile.analyticsIdentifier)
-        analyticsService.setUserValue(userProfile.teamInfo?.id, forKey: "team_team_id")
-        analyticsService.setUserValue(userProfile.teamInfo?.role, forKey: "team_user_type")
-        analyticsService.setUserValue(userProfile.teamInfo.map { String($0.size.logRound()) }, forKey: "team_team_size")
+        updateUserProfile(userProfile)
         analyticsService.beginSession()
 
         return AnalyticsSession(
             isSelfTeamMember: userProfile.teamInfo != nil,
             service: analyticsService
         )
     }
+
+    /// Disables tracking by ending the current session and clearing user data.
+    public func disableTracking() {
+        clearUserData()
+        analyticsService.endSession()
```

Comment:
It might be better to end the session first, then clear the user data.

Reply:
Agreed, fixed it here: https://github.com/wireapp/wire-ios/commit/d5ca76c32819a365b4035eb77982d7323c6b8a7f